app/assets/stylesheets/loan_documents.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the LoanDocuments controller here.
```

Comment:
@ericksonmc given the structure proposed in this PR, we don't need this file. Please remove it.